packages/instrumentation-socket.io/src/socket.io.ts

```diff
@@ -0,0 +1,170 @@
+import {
+    context,
+    setSpan,
+    Span,
+    SpanKind,
+    SpanStatusCode,
+    getSpan,
+    diag,
+    suppressInstrumentation,
+} from '@opentelemetry/api';
+import {
+    InstrumentationBase,
+    InstrumentationNodeModuleFile,
+    InstrumentationModuleDefinition,
+    InstrumentationNodeModuleDefinition,
+    isWrapped,
+} from '@opentelemetry/instrumentation';
+import { SemanticAttributes } from '@opentelemetry/semantic-conventions';
+import io from 'socket.io';
```

Comment:
you need to `import type`, otherwise this will throw in the absence of `socket.io` installation